app/controllers/provider/admin/cms/visit_portal_controller.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class Provider::Admin::CMS::VisitPortalController < Provider::Admin::CMS::BaseController
+  # Encrypt the CMS token under a temporary SSO token and redirect to the Developer Portal
+  def with_token
+    cms_token = current_account.settings.cms_token!
+    expires_at = Time.now.utc + 1.minute.to_i
```

Comment:
```suggestion
    expires_at = Time.now.utc + 1.minute
```

I think this `to_i` is not needed.

Reply:
Removed: https://github.com/3scale/porta/pull/3932/commits/301e48afa5ba2a35f69ba5bc1267abbdc4cc8b2c